stopcovid/utils/rollbar.py

```diff
@@ -0,0 +1,15 @@
+import os
+import sys
+
+import rollbar
+
+
+def _is_running_unit_tests():
+    return sys.argv[0].split(" ")[-1] == "unittest"
+
+
+def configure_rollbar():
+    if _is_running_unit_tests():
+        return
+
+    rollbar.init(os.environ.get("ROLLBAR_TOKEN"), environment=os.environ.get("STAGE"))
```

Comment:
I think these will land in rollbar just as "dev" and "prod" - might be helpful to prepend "dialog-engine-" to the environment